test/Controller/ArticleControllerTest.php

```diff
@@ -138,6 +138,13 @@ public function it_has_metadata()
         $this->assertSame('Article title', $crawler->filter('meta[property="og:title"]')->attr('content'));
         $this->assertSame('article', $crawler->filter('meta[property="og:type"]')->attr('content'));
         $this->assertSame('summary', $crawler->filter('meta[name="twitter:card"]')->attr('content'));
+        $this->assertEmpty($crawler->filter('meta[property="og:image"]'));
+        $this->assertSame('doi:10.7554/eLife.00001', $crawler->filter('meta[name="dc.identifier"]')->attr('content'));
```

Comment:
a bit heavy to test an identifier at the controller level, but the test was already in place so there is no test performance problem